snippets/cs.snippets

```diff
@@ -103,15 +103,15 @@ snippet for
 	}
 # foreach
 snippet fore
-	foreach (var ${1:entry} in ${2}) {
+	foreach (${1:var} ${2:entry} in ${3}) {
```

Comment:
Is it that you want to delete the `var` word sometimes? If so, can you also put the space in the `{}`s please.